subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt

```diff
@@ -71,12 +72,19 @@ public class InstrumentationTestsPlugin : Plugin<Project> {
 
         project.withAndroidModule { testedExtension ->
 
-            var pluginLevelInstrumentationArgs: Map<String, String> = emptyMap()
+            val dumpDir = File(extension.output, dumpDirName).apply {
+                mkdirs()
+            }
+
+            project.afterEvaluate {
```

Comment:
Do we really have to write files at any project configuration? Why not in the execution phase?

Reply:
there is no task to execute for this local run scenario
Android Studio test run doesn't actually run connectedCheck and the last known task is test and main apk's `install`

I don't like it either and it's not configuration cache friendly, but the only way i see it is to hide behind "test" flag as i did before

Reply:
Will you do it? Will you make it optional and explicit?

Reply:
yes